src/Reader/Filter/LikeMode.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Data\Reader\Filter;
+
+/**
+ * Like filter matching modes.
+ */
+enum LikeMode
+{
+    /**
+     * Field value contains the search value.
+     */
+    case CONTAINS;
+
+    /**
+     * Field value starts with the search value.
+     */
+    case STARTS_WITH;
+
+    /**
+     * Field value ends with the search value.
+     */
+    case ENDS_WITH;
+}
```

Comment:
It's recommended to use PascalCase in enums

```suggestion
enum LikeMode
{
    /**
     * Field value contains the search value.
     */
    case Contains;

    /**
     * Field value starts with the search value.
     */
    case StartsWith;

    /**
     * Field value ends with the search value.
     */
    case EndsWith;
}
```